### PR TITLE
package name change

### PR DESCRIPTION
**Pull request by nhsbsa-kaygo**
lowincomeschemeucd to nhslowincomeschemescrollytell

Change various places to reflect new package name.

### CODE REVIEW

File: .Rbuildignore [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+dev_history.R
+^dev$
+$run_dev.*
+^LICENSE\.md$
+^README\.Rmd$
+^CODE_OF_CONDUCT\.md$
+^doc$
+^Meta$
+^app\.R$
+^rsconnect$
```
Comment (ghost) [inline .Rbuildignore:13]: Think you need to add `.Rbuildignore` back into `.gitignore`

File: .gitignore [hunk 2]
```diff
@@ -11,5 +11,4 @@ man
 !/data
 /data/*
 !/data/.gitkeep
-rsconnect
-.Rbuildignore
\ No newline at end of file
+rsconnect
```
Comment (ghost) [inline .gitignore]: See previous comment

File: DESCRIPTION [hunk 3]
```diff
@@ -1,25 +1,9 @@
...
-             email = "kayoung.goffe1@nhs.net"),
-      person(given = "Adnan",
-             family = "Shroufi",
-             role = "ctb",
-             email = "adnan.shroufi@nhs.net"))
-Description: Low Income Scheme UCD.
-License: MIT + file LICENSE
+Authors@R: person('Adam', 'Ivison', email = 'adam.ivison@nhs.net', role = c('cre', 'aut'))
+Description: NHS Low Income Scheme Scrollytell.
+License: Apache License (>= 2)
```
Comment (ghost) [inline DESCRIPTION]: This needs changing back to have everyone?

File: DESCRIPTION [hunk 4]
```diff
@@ -38,15 +22,17 @@ Imports:
...
     Rcpp,
     sf,
-    udpipe,
-    tidytext,
-    textdata
+    attempt,
+    DT,
+    glue,
+    openxlsx,
+    markdown
```
Comment (ghost) [inline DESCRIPTION]: Don't need these?

File: README.Rmd [hunk 5]
```diff
@@ -0,0 +1,60 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+```{r, include = FALSE}
```
Comment (ghost) [inline README.Rmd]: Don't think we need this file

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
```
Comment (ghost) [inline CODE_OF_CONDUCT.md]: Don't think we need this file

File: README.md [hunk 7]
```diff
@@ -52,4 +52,8 @@ lowincomeschemeucd
 │   ├── utils_helpers.R                 # NHSBSA highcharter theme
 │   └── utils-pipe.R                    # %>% operator
```
Comment (ghost) [inline README.md]: Can we align the comments again?

File: README.md [hunk 8]
```diff
@@ -52,4 +52,8 @@ lowincomeschemeucd
...
 ├── README.md                           # Brief overview of the package
-```
\ No newline at end of file
+```
+
+## Code of Conduct
+
+Please note that the nhslowincomeschemescrollytell project is released with a [Contributor Code of Conduct](https://contributor-covenant.org/version/2/0/CODE_OF_CONDUCT.html). By contributing to this project, you agree to abide by its terms.
```
Comment (ghost) [inline README.md:59]: Don't need this

File: dev/01_start.R [hunk 9]
```diff
@@ -33,7 +33,7 @@ golem::set_golem_options()
 
 ## Create Common Files ----
 ## See ?usethis for more information
-usethis::use_mit_license( "NHS Business Services Authority" )  # You can set another license here
```
Comment (ghost) [inline dev/01_start.R:36]: See if you can add copyright holder as NHS Business Services Authority

**Review by ghost — CHANGES_REQUESTED** [review]

File: DESCRIPTION (hunk 4 above)
Comment (nhsbsa-kaygo) [reply DESCRIPTION]: I used openxls for reading income deprivation excel file.

markdown was added here as when i run dev_run.R file, it threw error not having markdown package.

**Review by nhsbsa-kaygo — COMMENTED** [review]
- DESCRIPTION package list: I used openxls and markdown was added as shiny threw error not finding the package.
- copyright holder is not a parameter for apache. instead you need to insert in LICENSE.md file (see line 182)

**Pull request merged**